src/ocean/util/app/DaemonApp.d

```diff
@@ -384,6 +402,12 @@ public abstract class DaemonApp : Application,
         this.config_ext.registerExtension(this.unix_socket_ext);
         this.registerExtension(this.unix_socket_ext);
 
+        if (settings.use_task_ext)
+        {
+            this.task_ext = new TaskExt();
+            this.config_ext.registerExtension(this.task_ext);
```

Comment:
I think you need `this.registerExtension(this.task_ext);` here too. (Though I forget what it does exactly.)